barretenberg/cpp/src/barretenberg/common/thread_utils.cpp

```diff
@@ -0,0 +1,20 @@
+#include "thread_utils.hpp"
+
+/**
```

Comment:
Everything should be in a namespace. in this case at least `namespace barretenberg`, but I think you should use  `namespace barretenberg::thread_utils` for clarity

Reply:
thread wasn't in a namespace, but this makes sense. I'll use barretenberg::thread_utils.